packages/data-portal-explore/src/components/CaseTable.tsx

```diff
@@ -16,6 +16,7 @@ import {
     sortByParticipantId,
 } from '../lib/dataTableHelpers';
 import { DataSchemaData, SchemaDataId } from '@htan/data-portal-schema';
+import { truncatedTableCell } from '../lib/dataTableHelpers';
```

Comment:
looks like we don't use this function in this component anymore, let's remove the `import` statement